docs/guides/user-guide/general-faqs.md

```diff
@@ -0,0 +1,189 @@
+
+
+### Can I change the TypeScript version in my project?
+
+The current TypeScript starter uses EngineBlock technology, which has a fixed TypeScript version. EngineBlock is great for lightweight frontends, but does not run Node.js itself and will be limited to things like TypeScript versioning.
+
+Webcontainer, however does run Node.js itself in the browser and does not have this limitation. You can simply `npm install` any TypeScript version as you would locally.
+
+## Classic Editor (WebContainer projects)
+
+### What is the Classic Editor for WebContainer projects?
+
+The Classic Editor utilizes Engineblock and WeBcontainer technologies to run. As the original StackBlitz interface, the Classic Editor has a file tree, an npm package manager, and a light dev server and preview pane that allows you to see your app as you code.
+
+Since WebContainer technology allows Node.js to run inside the browser, WebContainer projects will always have a terminal available.
+
+### How do I know if my project is WebContainers-based?
+
+All WebContainers projects will have a terminal in the project. You can find what environments are available for EngineBlock and WebContainers projects in our [docs](https://developer.stackblitz.com/guides/user-guide/available-environments).
+
+Any projects in `https://stackblitz.com/?starters=fullstack`, `..?starters=backend`, `..?starters=vite` will be WebContainer/Node projects with a terminal.
+
+### Can I change the Node version in my project?
+
+The current state of Codeflow does not support users being able to set their own Node version.
+
+### Why can't I see my terminal?
+
+Please note that the terminal is only available for WebContainer-based projects. You can use the keyboard binding `^` `` ` `` (the caret and the back quote keys) to toggle the terminal.
+
+### Can I run unit tests in StackBlitz?
+
+You can use [Jasmine](https://jasmine.github.io/), a unit testing tool for JavaScript.
+
+It is also possible to run Angular unit tests inside WebContainer-based projects, but not Engineblock-based projects. The easiest way to start is by navigating to node.new, which creates a WebContainer-based project automatically!
+
+### Can I develop ElectronJS apps on StackBlitz?
+
+Since Electron is a native app, this cannot be supported since StackBlitz runs entirely within the browser.
+
+## Codeflow Editor
+
+### What is the Codeflow Editor?
+
+The Codeflow Editor utilizes our Codeflow technology to bring VSC code into the browser. This editor is capable of running full development workflows, including pushing commits and creating pull requests to GitHub. This bypasses the need to clone a repository or install dependencies on your local machine. If you haven't already, you can check out Codeflow [here](https://stackblitz.com/codeflow)!
```

Comment:
rewrite as: "The Codeflow Editor brings VS Code into the browser."

Reply:
after "...creating pull requests to GitHub", add the features as a separate sentence.